lsp/src/main/java/org/aya/lsp/server/AyaService.java

```diff
@@ -272,6 +273,27 @@ private DiagnosticSeverity severityOf(@NotNull Problem problem) {
     });
   }
 
+  @Override public CompletableFuture<Hover> hover(HoverParams params) {
+    return CompletableFuture.supplyAsync(() -> {
+      var loadedFile = find(params.getTextDocument().getUri());
+      if (loadedFile == null) return null;
+      var doc = ComputeSignature.invoke(loadedFile, params.getPosition(), true);
+      if (doc.isEmpty()) return null;
+      return new Hover(new MarkupContent(MarkupKind.PLAINTEXT, Doc.stickySep(doc).commonRender()));
```

Comment:
Remember that `commonRender` sets the width limit to 80. The unlimited version is `debugRender`

Reply:
I forgot that 🤣